### DISCUSSION TITLE
How to enable scroll back after I clear the terminal?

### DISCUSSION THREAD

**Discussion by holaros** [ghostty-org/ghostty#4120] [Q&A]
Hi, I am new here (sorry if this is an amateur question -- I did try searching it up, I did not find any answer to it). Basically, the question -- if I clear screen using `clear`, how do I scroll back to it at some point later?

Thanks in advance!

**Comment by CoderJoshDK** [comment]
Ghostty properly does the clear instruction. Meaning you can't scroll back past it (it is actually other terminals that don't follow this instruction properly.)
If you want to be able bring your line to the top without clearing the scroll back, try `<C-l>` (`control + L`)
